chalk.c

```diff
@@ -1,11 +1,17 @@
 #include <unistd.h>
+#include <stdlib.h>  
 #include <termios.h>
+struct termios orig_termios;
 
+void disableRawMode(){
```

Comment:
For functions, the braces will start from the next line. This is an exception from the usual case of keeping them in the same line.

Reference: [Coding Style](https://bytesclub.github.io/docs/coding_style.pdf) Page 2, chapter 2